config/environments/development.rb

```diff
@@ -1,6 +1,15 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable = true
+    Bullet.alert = true
+    Bullet.bullet_logger = true
+    Bullet.console = true
+    Bullet.rails_logger = true
+    Bullet.add_footer = true
+  end
```

Comment:
These are auto-gen default configs. I'm open to changing them/don't have a strong opinion (yet).

Reply:
Seems fine to me. We can tweak these as we work with the gem and learn more about it and develop Opinions ™️.